client/devdocs/design/search-collection.jsx

```diff
@@ -64,7 +65,7 @@ const Collection = ( { children, filter, section = 'design', component } ) => {
 				unique={ !! component }
 				url={ exampleLink }
 			>
-				{ example }
+				{ component ? <PropsViewer component={ component } example={ example } /> : example }
```

Comment:
Why do we pass `example` to `<PropsViewer />` ? Seems we should just leave it where it was at and have `<PropsViewer />` be intentionally concerned only with... well, the props. Also helps avoid the `PureComponent` issue I mentioned in an earlier comment.